### ISSUE TITLE
May Meetup 2017

### ISSUE THREAD

**Issue by phoet** [rughh/planning#68]
10.05.2017 19:00

- [x] Location
- [x] [Event anlegen](http://hamburg.onruby.de/admin/)
- [x] [Talks / Format](https://github.com/rughh/planning/wiki/Speaker-Checklist)
- [x] Promo auf Twitter
- [x] [Push-Notification für iOS app senden](http://hamburg.onruby.de/admin/dashboard)
- [x] [Slides anpassen](https://docs.google.com/presentation/d/1qbNdAffMiYkPWUrUd5xeWHq6ge6vhIVQtuZomdiSorU/edit?usp=sharing)
- [x] [Door Sign anpassen](https://docs.google.com/presentation/d/1IQfnE3TbG3EU5zRnNKdG3kzxdYjT1ryWNWs4WWgpE04/edit?usp=sharing)
- [x] gather slides

avocadostore has offered to host the usergroup in may.

**Comment by janz93** [comment]
Awesome

**Comment by phoet** [comment]
additionally we received offers for hosting from sum.cumo: https://twitter.com/sumcumo/status/849625832347840512

and mindmatters: https://twitter.com/mindmatters/status/851765123421278208

**Comment by phoet** [comment]
unfortunately, avocado had to withdraw their offer. mindmatters was so nice to quickly accept to take us on this wednesday.

we are very low on the side of talks, if everybody could reach out to their network, that would be appreciated!

**Comment by phoet** [comment]
@halfbyte did you actually present something new on rug-b?

**Comment by halfbyte** [comment]
@phoet I just gave a quick sales pitch lightning talk. The kind people love so dearly (I made sure it was low-pitch, of course). I have two talks I could give: 

1. One on automations with a focus on the decisions we made with Depfu. I gave it at rubyconf.by and rubyconference.lt.
2. The JSConf talk which is about code aesthetics and has examples from a load of languages but is slightly tilted towards JavaScript, of course.

Both fit into roughly 30 minutes.

I could also talk about my Novation stuff, but that's so focused on Web Audio/Web MIDI and JavaScript that I'm not sure about the audience compatibility.

**Comment by phoet** [comment]
@halfbyte i'd vote for a none JS talk to keep it balanced 😄

**Comment by halfbyte** [comment]
@phoet Let me rephrase: It's tilted towards JS people. There's just one thing in there that's rally JS specific and a lot of people could relate to that. The good thing about that talk is that I gave it two days ago, so it's really well rehearsed. 

But given your prefs, I am fine with repeating the rubyconf.by talk. I'll prep for it. Interesting choice of speakers we'd have then. Damn, we need to fix this.

**Comment by phoet** [comment]
@halfbyte 😸
